torch/quantization/fx/quantization_patterns.py

```diff
@@ -277,6 +277,38 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 return quantizer.quantized_graph.create_node(
                     'call_function', torch.ops.quantized.conv2d, qconv_args, kwargs)
 
+@register_quant_pattern(torch.nn.ConvTranspose1d)
+@register_quant_pattern(torch.nn.ConvTranspose2d)
+class ConvTransposeQuantizeHandler(QuantizeHandler):
```

Comment:
can we use this: https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantization_patterns.py#L554
feel free to inherit as well if you want to ConvTranspose separate from the things already listed there.

Reply:
didn't know about that one, thanks